plugins/panels/model_evaluation/__init__.py

```
@@ -227,6 +231,33 @@
+        # use eval_key to delete the evaluation from dataset
+        eval_key = ctx.params.get("eval_key", None)
+
+        try:
+            ctx.dataset.delete_evaluation(eval_key)
+            store = self.get_store(ctx)
+            store.delete(eval_id)
+            ctx.ops.notify(
+                "Evaluation deleted successfully!",
+                variant="success",
+            )
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)